packages/stark-ui/testing/package.json

```diff
@@ -1,15 +1,12 @@
 {
-  "name": "@nationalbankbelgium/stark-ui/testing",
-  "types": "testing.d.ts",
-  "main": "../bundles/stark-ui-testing.umd.js",
-  "module": "../fesm5/testing.js",
-  "es2015": "../fesm2015/testing.js",
-  "esm5": "../esm5/testing/testing.js",
-  "esm2015": "../esm2015/testing/testing.js",
-  "fesm5": "../fesm5/testing.js",
-  "fesm2015": "../fesm2015/testing.js",
+  "ngPackage": {
```

Comment:
Can we also use an `ng-package.json` file for this as with `stark-ui` so we keep consistency along all our packages?